pkg/apis/networkoperator/v1/networkconfig_types.go

```diff
@@ -150,6 +150,10 @@ type OVNKubernetesConfig struct {
 	// The MTU to use for the tunnel interface
 	// Default is 1400
 	MTU *uint32 `json:"mtu,omitempty"`
+
+	// If true, then assume the nodes already have a running openvswitch.
+	// +optional
+	UseExternalOpenvswitch *bool `json:"useExternalOpenvswitch,omitempty"`
```

Comment:
do we support that for OVN?

Reply:
@danwinship Why do we support it for sdn? Do the same reasons apply for OVN?

Reply:
I think we support it for OpenShift SDN because some people are using DPDK and so need to be running the DPDK-ified OVS. But I wasn't really asking "*should* we support it with ovn-kubernetes?" so much as "*can* we support it with ovn-kubernetes?", or are OVN and ovn-kubernetes more tightly bound than OVS and OpenShift SDN are?

At the least it would presumably have to be "UseExternalOVN" rather than "UseExternalOpenvswitch", since if the user is going to replace OVS, they need to ensure they also run the version of OVN that goes with it, not necessarily the version of OVN that we're shipping.